sessions/2019/march/Session_28.md

```diff
@@ -0,0 +1,87 @@
+# SOFTWARE ENGINEERING IN JAVA
+
+## Session 28 (19/03/2019)
+
+- Reviewing sprint project (issue) tasks and solving problems & feedback
+- Reviewing main project tasks  & planning next Sprint
+
+## Notes
+
+#### Reviewing sprint project (issue) tasks and solving problems & feedback
+
+Public classes goes into separate files, to separate it quickly we can copy the class code and paste into the package directory and IntellJ Idea creates a class for us)
+
+Always that be possible we have to make attributes `final`. This keyword, means our object, field or property is in a fixed memory position and can be seen as a constant.
+
+There was an error when executing application: `Error: Spring Boot RestController Error: “No Converter Found for Return Value of Type”`.  Problem here were [getters and setters](https://dzone.com/articles/spring-boot-restcontroller-error-no-converter-foun)
+
+### Homework
+
+- To do the [Setup a database and read from it](https://github.com/javarb/wallet/issues/5) tasks as defined in the [project issue board](https://github.com/javarb/wallet/projects/2)
+
+####  Docker compose file that provisions a MySQL database.
+
+First we need to [install composer][1]:
```

Comment:
I would never get you to install [composer](https://getcomposer.org/) 😄

```suggestion
First we need to [install docker-compose][1]:
```

Reply:
ups....